packages/sdk/src/grpc/translation.ts

```diff
@@ -346,6 +351,8 @@ export function accountInfo(acc: v2.AccountInfo): v1.AccountInfo {
         total: CcdAmount.fromProto(unwrap(acc.schedule?.total)),
         schedule: unwrap(acc.schedule?.schedules).map(trRelease),
     };
+    const cooldowns = acc.cooldowns.map(transCooldown);
+    const availableBalance = CcdAmount.fromProto(unwrap(acc.availableBalance));
```

Comment:
**Suggestion:** The `unwrap` here will mean this query fails when used against concordium-nodes prior to `7.0.0`.
So we should probably document the minimum support node version somewhere.

Reply:
Yes, this will be merged into the release branch for version 8 of the SDK, and I think the strategy should be to release it in stages through npm tags:

1. alpha = local/stagenet node 7.0 release
2. beta = testnet node 7.0 release
3. latest = mainnet node 7.0 release

I'll figure out where to document the minimum supported node version as well.

Reply:
I added a release schedule to https://github.com/Concordium/concordium-node-sdk-js/pull/385 which is a release PR for the target branch.